### PR TITLE
New start arrows and small UI/UX fixes

### PR DESCRIPTION
**Pull request by CSchank**
To be consistent with 2FA3 and other classes, this implements start arrows, meaning that bold is now only for the current state when simulating. This should make a lot more sense to users, and help them learn the content without distractions such as these. Also implements some fixes for a few issues: 

- eliminating autosave calls when not logged in
- minor tweaks to make text boxes nicer
- the ability to use the Enter key to toggle between editing and not editing of state/transition labels. This is my new favourite way to edit! Select the thing you want to edit and press enter.
- adds a link to our GitHub on the welcome screen (not sure if the link works though!)

@Necried please look this over and then I will put it up for tomorrow's demo.

### CODE REVIEW

File: src/SaveLoad.elm [hunk 1]
```diff
@@ -380,15 +380,15 @@ subscriptions model =
...
+            ++ (case ( model.machineData, model.loginState ) of
+                    ( MachineCreated, LoggedIn _ _ ) ->
                         if model.unsavedChanges then
                             [ Time.every 5000 (MachineCreatedMsg << AutoSave) ]
 
                         else
                             []
 
-                    MachineNotCreated ->
+                    ( MachineNotCreated, _ ) ->
```
Comment (Necried) [inline src/SaveLoad.elm]: This is causing an error (incomplete cases). Changing this to `(_, _)` fixes the issue for me

Comment (CSchank) [reply src/SaveLoad.elm]: Yeah, my bad! I realized this and fixed in the newest commit. Also, added a pen icon to make it clear you can edit the name of the machine.

Comment (CSchank) [reply src/SaveLoad.elm]: This why we have CI! Apparently Travis is making it more difficult for open source projects to get free builds, so we might have to switch....

Comment (Necried) [reply src/SaveLoad.elm]: Good to know; we can look at alternatives

**Review by Necried — APPROVED** [review]

**Comment by CSchank** [comment]
I'm not sure why this is saying it didn't pass, on Travis it's green. So I'll merge regardless.

**Pull request merged**